cpp/kiss_icp/pipeline/KissICP.hpp

```diff
@@ -67,6 +67,7 @@ class KissICP {
     double GetAdaptiveThreshold();
     Sophus::SE3d GetPredictionModel() const;
     bool HasMoved();
+    void Reset(Eigen::Quaterniond, Eigen::Vector3d);
```

Comment:
Please use Sophus::SE3d as argument. Quaternion&Vector is more of a ROS thing and I will like to keep the cpp API Sophus only.

Reply:
Sure.

Reply:
Solved